tutoraspects/templates/aspects/apps/superset/pythonpath/create_assets.py

```diff
@@ -237,6 +238,15 @@ def update_dashboard_roles(roles):
             dashboard.owners = owners
         db.session.commit()
 
+        embedded_dashboard = db.session.query(EmbeddedDashboard).filter_by(dashboard_id=dashboard.id).first()
```

Comment:
This will allow any dashboard with roles assigned to it to be embedded, right? That may not be desirable, I think I would make this its own method with a separate flag for whether the dashboard should be able to be embedded.

Reply:
What do you think of this implementation?

People will be able to set up an embeddable ID for every Aspects dashboard as they wish.